rust/processor/migrations/2024-05-21-221101_add_royalty_v1/up.sql

```diff
@@ -0,0 +1,23 @@
+-- Your SQL goes here
+CREATE TABLE token_royalty (
+    transaction_version BIGINT NOT NULL,
+    token_data_id VARCHAR(66) NOT NULL,
+    payee_address VARCHAR(66) NOT NULL,
+    royalty_points_numerator NUMERIC NOT NULL,
+    royalty_points_denominator NUMERIC NOT NULL,
+    token_standard TEXT NOT NULL,
+    inserted_at TIMESTAMP NOT NULL DEFAULT NOW(),
+    PRIMARY KEY (transaction_version, token_data_id) 
+);
+
+CREATE TABLE current_token_royalty (
+    token_data_id VARCHAR(66) NOT NULL,
```

Comment:
I forgot why we don't have a unique id for the token.. but using hash value along as primary key can be dangerous since collision may happen.

Reply:
That's a good point. Including `creator_address` in the primary key can help with collision. Let me add that.

Reply:
@larry-aptos explained to me how collision can happen, but i forgot the reason cc @grao1991

Reply:
hash id is not guaranteed to be unique. It may be a problem when two different entities have the same hash value.

it seems ahash uses 64bit and considering 10million items, we get this

![image](https://github.com/aptos-labs/aptos-indexer-processors/assets/112209412/68454079-bdff-4b85-898c-553db4da2769)

If we consider the number of items(i.e. token) can go up to 10m, we should take the probability of collision into account and add a field to make pk unique, otherwise, we should be fine.

Reply:
wait what? these hashes aren't randomly generated though so can collision still happen? I don't think that creator address is necessary b/c it's part of the hash.

Reply:
Also the tokens tables don't have this level of protection so we'll end up with issues anyway.